### DISCUSSION TITLE
LazyVim has a weird color scheme at installation

### DISCUSSION THREAD

**Discussion by sharathlingam** [LazyVim/LazyVim#3103] [Q&A] [answered]
I downloaded LazyVim(LV) in my macbook air M1 for the first time and I uninstalled it which worked as expected for me but since my second installation LV has a weird color scheme after cloning and executing the command `nvim` inside the ~/.config/nvim directory. I did the neovim installation using `brew install neovim` then I removed all the previous LV `~/.config/nvim` folder whenever I tried to reinstall it again I am getting this weird theme.
Here is my log:
```
Last login: Sun May  5 02:48:48 on ttys000
➜  ~ vim
➜  ~ nvim
➜  ~ mv ~/.config/nvim{,.bak}

mv: rename /Users/sharathlingam_s/.config/nvim to /Users/sharathlingam_s/.config/nvim.bak: No such file or directory
➜  ~ mv ~/.local/share/nvim{,.bak}
mv ~/.local/state/nvim{,.bak}
mv ~/.cache/nvim{,.bak}
mv: rename /Users/sharathlingam_s/.local/share/nvim to /Users/sharathlingam_s/.local/share/nvim.bak/nvim: Directory not empty
mv: rename /Users/sharathlingam_s/.cache/nvim to /Users/sharathlingam_s/.cache/nvim.bak/nvim: Directory not empty
➜  ~ git clone https://github.com/LazyVim/starter ~/.config/nvim
Cloning into '/Users/sharathlingam_s/.config/nvim'...
remote: Enumerating objects: 198, done.
remote: Counting objects: 100% (5/5), done.
remote: Compressing objects: 100% (5/5), done.
remote: Total 198 (delta 0), reused 2 (delta 0), pack-reused 193
Receiving objects: 100% (198/198), 63.57 KiB | 2.05 MiB/s, done.
Resolving deltas: 100% (74/74), done.
➜  ~ rm -rf ~/.config/nvim/.git
➜  ~ nvim
```

And here is how I am getting the resulting LV dashboard,

![image](https://github.com/LazyVim/LazyVim/assets/111525679/4ef0677a-bec8-4619-99b2-60f34cfb97b1)

<img width="1440" alt="image" src="https://github.com/LazyVim/LazyVim/assets/111525679/f4dbf081-e387-4a51-8314-ac09a3ec54c1">

<img width="1440" alt="image" src="https://github.com/LazyVim/LazyVim/assets/111525679/e509a928-ae20-4f9c-90c3-6faf5efd4a0e">

I tried thrice and got the same result. I tried as fourth time watching https://www.youtube.com/watch?v=V070Zmvx9AM this tutorial still got the same screens. Please help me with this issue. Thank you so much for your response in advance.

P.S. Editted: (This is how my neovim editor looks like and I installed it using brew. Idk why it is having weird colors too.)
<img width="1440" alt="image" src="https://github.com/LazyVim/LazyVim/assets/111525679/bac4594e-a3cd-436e-8b8a-9d94bd75b201">

**Comment by dpetka2001** [comment, marked as answer]
Try a different terminal (kitty, wezterm, iterm2) and see if that fixes your issue.

**Reply by sharathlingam** [reply]
@dpetka2001 Thanks alot it fixed my issue
